### ISSUE TITLE
2.5.1 / 2.4.5 patch releases plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#2019]
## 2.5.1
- #1990
- #1948 
- #2003 
- #2004
- #1979 
- #2021

## 2.4.5
- #1990
- #2003
- #2004  
- #1979 
- #2021

**Comment by joeparsons** [comment]
## Pick lists

### 2.5.1
```
git cherry-pick 4f8f0f2e4a1edf5a4af3a79adc62e393b449a8ab
git cherry-pick 0eb1cdeb26a418bf33f4b6ef76d18343456d8d31
git cherry-pick a8b7b32ce877918bb4913f442790caa5e2911a3f
git cherry-pick 0f3eaf0b8771a8190affd552c6378ecfa73b865d
git cherry-pick f265e45a5f2ee0f390b4ef6cc8a1979f09662d75
git cherry-pick 1a9ff57cb8868ed9804fe11fb0e4a3563e9c8712
```

### 2.4.5
```
git cherry-pick 4f8f0f2e4a1edf5a4af3a79adc62e393b449a8ab
git cherry-pick a8b7b32ce877918bb4913f442790caa5e2911a3f
git cherry-pick 0f3eaf0b8771a8190affd552c6378ecfa73b865d
git cherry-pick f265e45a5f2ee0f390b4ef6cc8a1979f09662d75
git cherry-pick 1a9ff57cb8868ed9804fe11fb0e4a3563e9c8712
```

**Comment by joeparsons** [comment]
## Release notes snippets

### 2.5.1
`git log 2.5.0..HEAD --pretty='format:- %s'`
```
- Closes #2016: Update drupal/draggableviews requirement from 2.0.1 to 2.1.1 (#2021)
- Update drupal/core-recommended requirement from 9.4.7 to 9.4.8 (#1979)
- Update PR template formatting (#2004)
- Closes #2002: Fix memory leak in core to improve stability of migrations. (#2003)
- Closes #1913, Closes #1997 Update az_custom_menu_block title style and fix demo content where the card deck page shows menu blocks. (#1948)
- Closes #408, Closes #1449 Remove reorder people block and add contextual links. (#1990)
- Back to dev.
```

### 2.4.5
`git log 2.4.4..HEAD --pretty='format:- %s'`
```
- Closes #2016: Update drupal/draggableviews requirement from 2.0.1 to 2.1.1 (#2021)
- Update drupal/core-recommended requirement from 9.4.7 to 9.4.8 (#1979)
- Update PR template formatting (#2004)
- Closes #2002: Fix memory leak in core to improve stability of migrations. (#2003)
- Closes #408, Closes #1449 Remove reorder people block and add contextual links. (#1990)
- Back to dev.
```

**Comment by joeparsons** [comment]
Releases created:
- https://github.com/az-digital/az_quickstart/releases/tag/2.5.1
- https://github.com/az-digital/az_quickstart/releases/tag/2.4.5
